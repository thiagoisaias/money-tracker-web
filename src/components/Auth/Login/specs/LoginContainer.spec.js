import React from "react";
import ReactDOM from "react-dom";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import { LoginContainer } from "../LoginContainer";

describe("Login Container", () => {
  const mockProps = {
    error: null,
    onClearAuthError: jest.fn(),
    onLogin: jest.fn(),
    isLoading: false,
    history: {}
  };
  const component = shallow(<LoginContainer {...mockProps} />);

  it("renders properly", () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
