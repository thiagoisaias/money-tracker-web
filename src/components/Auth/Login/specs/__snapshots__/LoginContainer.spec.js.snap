// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Container renders properly 1`] = `
<Component
  error={null}
  isLoading={false}
  onLogin={[Function]}
  onSubmitData={[Function]}
/>
`;
